django_project/ford3/forms/user_management_form.py

```diff
@@ -0,0 +1,84 @@
+from django import forms
+from django.core.validators import validate_email
+from django.contrib.auth import password_validation
+from django.utils.translation import gettext_lazy as _
+
+
+class UserManagementForm(forms.Form):
+    group = forms.CharField(
+        widget=forms.HiddenInput,
+        required=True)
+    email_confirmed = forms.BooleanField(
+        widget=forms.CheckboxInput,
+        required=False)
+    email = forms.EmailField(
+        widget=forms.EmailInput,
+        required=True)
+    error_messages = {
+        'invalid_email': _("Invalid email"),
+    }
+
+    def clean_email(self):
+        email = self.cleaned_data['email']
+        validate_email(email)
+        return email
+
+
+class UserManagementPasswordResetForm(forms.Form):
```

Comment:
should be UserManagementAccountActivationForm

Reply:
yes, you are right. the name is misleading. i'll change the form name